week2/index.js

```diff
@@ -1,65 +1,150 @@
+// require filesystem library because we are going to write and read files
 var fs = require('fs');
 
-var options = process.argv.slice(2);
+//this object will contain the todo's info and will be written to the todosFile 
+// I have chosen this to be a JSON because we can use the JSON proberities on it
+let todosFile = JSON.parse(fs.readFileSync('todo.json','UTF-8'));
+console.log(todosFile);
 
-var command = options[0];
+//shorten the arguments array to the arguments we need
+//take out the first two arguments which are the node path and the file path
+let options = process.argv.slice(2);
 
+//check which command we have got from the user, which is the first argument in our new arguments array
+let command = options[0];
+
+//evaluate which action to take based on the command the user entered.
 switch (command) {
-  case 'help':
-  default:
-    showHelp();
-    break;
-  case 'list':
-    listTodos();
-    break;
+    //if the user enter the word help as command
+    //we will display an introduction to all the commands available
+    //We will do it,through calling the functions showHelp
+    case 'help':
```

Comment:
What would happen if you moved this down to just above the `default` case and removed the `break`?